open_fdd/tests/desktop/test_desktop_bridge.py

```
@@ -469,6 +469,283 @@ def test_data_model_health_summary_endpoint_shape() -> None:
+      path: tiny.csv
+      source: csv
+    equipment:
+      name: RTU
+      type: Air_Handling_Unit
+    brick_mappings:
+      - external_id: sat
+        brick_type: Supply_Air_Temperature_Sensor
+""".strip(),
+        encoding="utf-8",
+    )
+    monkeypatch.setattr(gateway_server, "_examples_pack_root", lambda: ex.resolve())
```

Ruff (0.15.12)

[error] Undefined name `gateway_server`

(F821)